stl/src/awint.hpp

```diff
@@ -16,17 +16,7 @@ _CRT_BEGIN_C_HEADER
 _CRTIMP2 BOOL __cdecl __crtIsPackagedApp();
 #endif // !defined(_CRT_WINDOWS) && !defined(UNDOCKED_WINDOWS_UCRT)
 
-#if _STL_WIN32_WINNT >= _WIN32_WINNT_WS03
-
-#define __crtFlsAlloc(lpCallback) FlsAlloc(lpCallback)
-
-#define __crtFlsFree(dwFlsIndex) FlsFree(dwFlsIndex)
-
-#define __crtFlsGetValue(dwFlsIndex) FlsGetValue(dwFlsIndex)
-
-#define __crtFlsSetValue(dwFlsIndex, lpFlsData) FlsSetValue(dwFlsIndex, lpFlsData)
-
-#else // _STL_WIN32_WINNT >= _WIN32_WINNT_WS03
+#if _STL_WIN32_WINNT < _WIN32_WINNT_WS03

```

Comment:
Looks like this macro can be completely eliminated now

Reply:
Yes! Thought about this a bit more, and understood that only `.cpp` implementation needs to be preserved, not declaration.